README.md

```diff
@@ -11,16 +11,19 @@ These core user-facing software include:
 - [JupyterHub](https://github.com/jupyterhub/jupyterhub)
 
 > **Important notice**
-> As of August 2022 this software **is not accessible.**
+> As of January 2023 this software **is not accessible.**
```

Comment:
I feel like this is related to a prior discussion I am forgetting. but was January 2023 chosen to move this date up for more recency? I think August 2022 overlapped with a round of review and was listed to describe that.

I don't think this is a dealbreaker since the accessibility statement also lists the reviews. But I'm not sure if we did a review in January 2023 I'm forgetting about.

Reply:
I think there were some minor wording changes in January (think I saw the commit history) but I can always revert back to Aug to align with the audit.